pkg/registry/core/serviceaccount/storage/token_test.go

```diff
@@ -121,9 +118,6 @@ func TestCreate_Token_WithExpiryCap(t *testing.T) {
 			// add the namespace to the context as it is required
 			ctx = request.WithNamespace(ctx, serviceAccount.Namespace)
 
-			// Enable ExternalServiceAccountTokenSigner feature
-			featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.ExternalServiceAccountTokenSigner, true)
```

Comment:
revert this change, continuing to set the feature gate to true explicitly is fine (and keeps this test passing if we have to toggle the gate back off by default for some reason)